### PR TITLE
Allow to set from address

### PR DESCRIPTION
**Pull request by Infern1**
### Changes proposed in this pull request:
- To allow change from address, for example php mailing this need to be set
-
-

Fixes # .

Checklist:
- [ ] docs up-to-date

### CODE REVIEW

File: provision-base.sh [hunk 1]
```diff
@@ -60,7 +60,8 @@ install_ssmtp()
 	sed -e "/^root=/ s/postmaster/$TOASTER_ADMIN_EMAIL/" \
 		-e "/^mailhub=/ s/=mail/=haraka/" \
 		-e "/^rewriteDomain=/ s/=\$/=$TOASTER_MAIL_DOMAIN/" \
-		"$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
+                -i '/^#FromLineOverride=YES/ s/#//' \
+                "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
 		> "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf" || exit
```
Comment (msimerson) [inline provision-base.sh]: Did you test this on the CLI?  I doesn't look to me like it's going to do what you want it to. Also, you appear to have `expandtab` set in vim still. ;-)

Comment (Infern1) [inline provision-base.sh]: At least have this setting in production and does what it should do.
If I don't set it, mail is send as "User &" if I set this to yes: "wordpress@<domainname>"

Fixed the expandtab (it was twice in my .vimrc

File: provision-base.sh [hunk 2]
```diff
@@ -60,6 +60,7 @@ install_ssmtp()
 	sed -e "/^root=/ s/postmaster/$TOASTER_ADMIN_EMAIL/" \
 		-e "/^mailhub=/ s/=mail/=haraka/" \
 		-e "/^rewriteDomain=/ s/=\$/=$TOASTER_MAIL_DOMAIN/" \
+		-i '/^#FromLineOverride=YES/ s/#//' \
 		"$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
```
Comment (msimerson) [inline provision-base.sh]: are you **sure** that you want `-i` there?

Comment (Infern1) [inline provision-base.sh]: no.. :( copy/pasted... not 100% knowledgeable about sed...

**Pull request merged**